index.js

```diff
@@ -18,27 +17,54 @@ function createOnResponseHandler(action) {
 }
 
 app.post('/', upload.single('thumb'), function (req, res, next) {
-  var payload = JSON.parse(req.body.payload);
+  const payload = JSON.parse(req.body.payload);
 
-  var validUser   = payload.Account.id === 1 || payload.Account.title === process.env.USERNAME;
-  var validPlayer = payload.Player.uuid === process.env.PLAYER;
+  const isValidUser = (
+    payload.Account.id === 1 ||
```

Comment:
@elan how is `payload.Account.id === 1` different from `payload.owner`?

Reply:
I don't have a good answer for this one; I just shuffled the bits around.

Reply:
From [the documentation](https://support.plex.tv/articles/115002267687-webhooks/) ~ "The user and owner flags indicate whether the event is sent because the user has a webhook configured (user flag is set) or because the server owner has a webhook set (the owner flag is set)" (separate from the user the event is _about_).